LittlePoops/little-poops-01.py

```diff
@@ -0,0 +1,35 @@
+# Write a function that takes a positive integer as an argument &
+# returns a Boolean indicating whether the argument is a power of three.
+
+# Hint: you might want to use recursion for this, which we haven't talked about yet. Exciting!
+
+def threepower(x,p):
+    p = int(p)
```

Comment:
I strongly recommend that you assume that the parameters for your function are formatted properly, unless the point of the function is to convert values from one type to another. Not only does that keep your function code clean and readable, it also means that your data validation can happen closer to when it was entered by the user, which helps you deal with any errors.